src/components/personalize/personalize.styles.js

```diff
@@ -7,7 +7,11 @@ function personalizeStyles(colors) {
   return `
 
 .is-personalizable ::selection {
-  background: ${colors.selection} !important
+  background: ${colors.selection} !important;
```

Comment:
```suggestion
  background: ${colors.darker} !important;
```

Not sure if this will work better if you just change it for all elements inside a is-personalizable area?

But what is wierd is i dont see it working at all on this page?
http://localhost:4000/components/personalize/example-form.html

Reply:
@tmcconechy @RiyaTayal20 I think we're still missing a CSS rule for `::selection` to be applied to headers without personalization rules on this PR

Reply:
Probably need one here somewhere: https://github.com/infor-design/enterprise/blob/main/src/components/header/_header.scss#L15

Reply:
Oh this is what I pushed on Friday so you would be able to see what was going on @EdwardCoyle this issue hasn't been finalized yet.